### ISSUE TITLE
Rename the velodyne package names to velodyne_awf

### ISSUE THREAD

**Issue by xmfcx** [autowarefoundation/velodyne_awf#1]
Following the discussions in https://github.com/autowarefoundation/autoware/issues/3222

Following packages should be suffix with `_awf`
```
- velodyne
- velodyne_driver
- velodyne_laserscan
- velodyne_msgs
- velodyne_pointcloud
```

**Comment by esteve** [comment]
I'd prefix them instead (`awf_velodyne_xxx`), easier to see which packages have been forked by the AWF. @xmfcx Can you assign this issue to me? Thanks.

**Comment by esteve** [comment]
> We won't have many packages similar to this but even then, I think suffix is better to keep lexicographical order. The repository name was `velodyne_vls` too.

Yes, both upstream and the fork were named `velodyne_vls` because they added support for the VLS line of velodyne sensors, hence the suffix. In our case, we want to signal that they're a fork made by the AWF organization.

> Do you have any other specific reason to have it as a prefix?

It makes it clearer that it's the AWF fork when we generate the Debian packages.

I prefer either a prefix or a suffix at the end, adding `awf` in the middle is in my opinion even more confusing.

**Comment by xmfcx** [comment]
I don't perceive it as adding it in the middle, I see it just as renaming `velodyne` to `velodyne_awf` to keep the lexico-order. If you have strong opinions to keep them as `awf_velodyne` and many packages already have the same structure in naming, let's follow it then.

So, how deep will this refactoring be? Will packages, executables, library names all be renamed?

**Comment by esteve** [comment]
> So, how deep will this refactoring be? Will packages, executables, library names all be renamed?

For now I think only renaming the package is necessary, the rest can stay the same. The problems with generating the Debian packages were the name of the package and the mixup with the versions:

 - `CHANGELOG` says it's 1.3.0 (https://github.com/tier4/velodyne_vls/blob/tier4/universe/velodyne_driver/CHANGELOG.rst?plain=1#L4)
 - `package.xml` says it's 0.2.0 (https://github.com/tier4/velodyne_vls/blob/tier4/universe/velodyne_driver/package.xml#L4)
 - latest tag is v0.11.0beta (https://github.com/tier4/velodyne_vls/releases/tag/v0.11.0beta)

So we'll need to do some cleanup with the versions as well.

We can just transfer the `velodyne_vls` repository to the `autowarefoundation` organization as is, and only need to rename the packages inside the repository in their `package.xml` files.

**Comment by miursh** [comment]
As I told @mitsudome-r, I totally agree with renaming the packages as separate from upstream. Either prefix or suffix is okay for me.
I also agree with cleaning up the version information in the repo since its not managed at all. Just putting the new version like v0.1 or something might be okay.

**Comment by drwnz** [comment]
While there are arguments for both prefiix and suffix, I would tend to agree with @esteve that the prefix keeps in line with the current naming of packages.
Re-naming the packages is good but I would also suggest adding the prefix to the repo as well. Reason being, that it just makes it clearer which version you are building - the differences between TIER IV's `velodyne_vls` and the upstream one is so large that a different repo name is justified (we'll probably never be able to easily merge into upstream).

**Comment by xmfcx** [comment]
I've updated the issue and the repository's names to match the prefix format. @mitsudome-r will discuss with the internal TIER IV team and write down his thoughts about whether to transfer, archive or keep the [tier4/velodyne_vls](https://github.com/tier4/velodyne_vls) repository.

**Comment by xmfcx** [comment]
Actually I will remove it now, otherwise it might prevent the other repo from being renamed.
